application-analytics-default/src/main/java/com/xwiki/analytics/script/AnalyticsScriptService.java

```diff
@@ -66,4 +70,14 @@ public JsonNode getMatomoRequestResult(Map<String, String> parameters, Map<Strin
             throw new RuntimeException(String.format("Failed to get data for [%s]", jsonNormaliserHint), e);
         }
     }
+
+    /**
+     *
+     * @return the analytics configuration.
+     * @since 1.0
+     */
+    public AnalyticsConfiguration getConfiguration()
```

Comment:
Missing [@Unstable](https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#H40UnstableAnnotation) annotation, which is needed for every new public API added. And I realized you don't need the `since` in the comment, since 1.0 it's the first version. Starting with 1.1 you will need to add it

Reply:
Unstable is still missing